Unix/regress

```diff
@@ -278,7 +278,7 @@ fi
 ##
 
 rm -rf $prefix
-./configure --prefix=$prefix --outputdirname=$current_output $options
+./configure --enable-werror --prefix=$prefix --outputdirname=$current_output $options
```

Comment:
--enable-microsoft is how the nightly builds will build OMI. So the question is, should the nightly builds use --enable-werror, which would guarantee no regressions in this area?